client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,103 @@
-function RestaurantPreview({ restaurant }) {
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Link } from "react-router-dom";
+
+const classes = {
+  previewContainer: {
+    height: "120px",
+    width: "100%",
+    backgroundColor: "rgb(224,224,224)",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "75%",
+    width: "100px",
+    margin: "1rem",
+    objectFit: "cover",
+  },
+  detailsContainer: {
+    marginLeft: "0.25rem",
```

Comment:
so this is the container that shows the restaurant info...
We can fix the category position bug two ways:
1. Easiest: slap a `position absolute` on it relative to the bottom of this container, and make this container `height 100%`
2. Flexbox: slap `display flex` and `flexDirection column` so it aligns downwards
then after that, `marginTop auto` on the categories div will probably fix it.